_commands/basics/who.md

```diff
@@ -3,22 +3,65 @@
 
 who
 --
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+## Description
+Displays list of all users currently logged on 
 
 <!-- minimal example -->
+### Example Output 
 ~~~ bash
-*short* example of command usage and output
+$ Username console  Oct 12 08:20 
+$ Username ttys000  Oct 20 09:35 
 ~~~
 
-<!--more-->
```

Comment:
This line needs to be brought back in to fix the reference page formatting errors.

This image shows the formatting errors for the reference page.
Anything between the `who` command at the top and the `<!--more-->` comment will be displayed on the reference page.

![screen shot 2017-11-02 at 11 26 52 pm](https://user-images.githubusercontent.com/8933413/32359819-f36811c2-c026-11e7-9a0f-b44474e7aa51.png)